cli/tests/pcluster/config/test_section_cluster.py

```diff
@@ -501,6 +513,246 @@ def test_sit_cluster_param_from_file(
+        ("additional_cfn_template", "http://test", "http://test", None),
+        ("additional_cfn_template", "s3://test/test2", "s3://test/test2", None),
+        ("additional_cfn_template", "NONE", "NONE", None),
+        ("tags", None, {}, None),
+        ("tags", "", {}, None),
+        ("tags", "{}", {}, None),
+        ("tags", "{'test': 'test'}", {"test": "test"}, None),
+        ("tags", "{'test': 'test'", None, "Error parsing JSON parameter"),
+        ("enable_intel_hpc_platform", None, False, None),
+        ("enable_intel_hpc_platform", "", None, "must be a Boolean"),
+        ("enable_intel_hpc_platform", "NONE", None, "must be a Boolean"),
+        ("enable_intel_hpc_platform", "true", True, None),
+        ("enable_intel_hpc_platform", "false", False, None),
+        # TODO add regex for custom_chef_cookbook
+        ("custom_chef_cookbook", None, None, None),
+        ("custom_chef_cookbook", "", "", None),
+        ("custom_chef_cookbook", "fake_value", "fake_value", None),
+        ("custom_chef_cookbook", "http://test", "http://test", None),
+        ("custom_chef_cookbook", "s3://test/test2", "s3://test/test2", None),
+        ("custom_chef_cookbook", "NONE", "NONE", None),
+        # Settings
+        ("scaling_settings", "test1", None, "Section .* not found in the config file"),
+        ("vpc_settings", "test1", None, "Section .* not found in the config file"),
+        ("ebs_settings", "test1", None, "Section .* not found in the config file"),
+        ("ebs_settings", "test1,test2", None, "Section .* not found in the config file"),
+        ("ebs_settings", "test1, test2", None, "Section .* not found in the config file"),
+        ("efs_settings", "test1", None, "Section .* not found in the config file"),
+        ("raid_settings", "test1", None, "Section .* not found in the config file"),
+        ("fsx_settings", "test1", None, "Section .* not found in the config file"),
+        ("queue_settings", "test1", None, "Section .* not found in the config file"),
+        ("queue_settings", "test1,test2", None, "Section .* not found in the config file"),
+        (
+            "queue_settings",
+            "test1,    LongestQueueNameThatIsPossible,    LongestQueueNameThatIsPossible",
+            None,
+            "Section .* not found in the config file",
+        ),
+        (
+            "queue_settings",
+            "test1,2test",
+            None,
+            (
+                "ERROR: The configuration parameter 'queue_settings' has an invalid value '.*'\nAllowed values are: "
+                "\\^\\[a-zA-Z\\]\\[a-zA-Z0-9-\\]\\{0,29\\}\\(,\\\\s\\*\\[a-zA-Z\\]\\[a-zA-Z0-9-\\]\\{0,29\\}\\)\\*\\$"
+            ),
+        ),
+        (
+            "queue_settings",
+            "LongQueueNameJustOneCharTooLong",
+            None,
+            (
+                "ERROR: The configuration parameter 'queue_settings' has an invalid value '.*'\nAllowed values are: "
```

Comment:
minor you can avoid to put the regex, you can just write: 
`ERROR: The configuration parameter 'queue_settings' has an invalid value .*`
or `.*  'queue_settings' has an invalid value .*`

In this way we don't need to change this failing test when changing the regex

Reply:
Done